stack.c

```diff
@@ -5,55 +5,47 @@
 #include "stack.h"
 
 void push(Stack* stack, stackType data) {
-    Node* newNode = malloc(sizeof(Node));
-    if(newNode == NULL) {
+    if(stack->top == MAX_SIZE) {
         printf("Stack overflow while push()");
-        return;
+        return ;
     }
-    newNode->data = data;
-    newNode->next = stack->top;
-    stack->top = newNode;
+    stack->top = stack->top + 1;
+    stack->array[stack->top] = data;
 }
 
 stackType pop(Stack* stack) {
-    if(stack->top == NULL) {
+    if(stack->top == -1) {
         printf("Stack underflow while pop\n");
         return -1;
     }
-    Node* next = stack->top->next;
-    stackType data = stack->top->data;
-    free(stack->top);
-    stack->top = next;
-    return data;
+    stack->top == -1;
```

Comment:
Did you run your code before submitting?

Reply:
yes sir it didnt give the proper output 
I didnt know the mistake